### ISSUE TITLE
Should fastLLaMa support more than just Python?

### ISSUE THREAD

**Issue by PotatoSpudowski** [PotatoSpudowski/fastLLaMa#13]
Hi, Would it be interesting to get fastLLaMa to support more than just Python? 

![Screenshot 2023-03-24 at 11 19 08 AM](https://user-images.githubusercontent.com/29813400/227435633-849e534a-d507-4cb4-b869-fdb829934f50.png)

Thoughts @amitsingh19975 @raldebsi ? 🤔

**Comment by raldebsi** [comment]
That would be cool since the concept of bridging is nice, but in pretty sure by now others have already begun work on other bridges so maybe it would be better to make a new repo that combines all the works?

**Comment by dahlej** [comment]
yes! Lets do this. we need to create a folder called bindings with one for each language below.

**Comment by PotatoSpudowski** [comment]
@dahlej We have created a folder called interfaces where new languages can be added! 

List of languages to consider 
- [ ] Rust
- [ ] NodeJS/WebAssembly
- [ ] Java
- [ ] Go

If anyone else feels like we need to support any other languages please do request it!
